client/tso_dispatcher.go

```diff
@@ -115,6 +86,17 @@ func (c *tsoClient) dispatchRequest(request *tsoRequest) (bool, error) {
 	return false, nil
 }
 
+func (c *tsoClient) closeTSODispatcher() {
```

Comment:
Shall we move the tsoClient-related function to tso_client.go?

Reply:
Currently, because all data structures are inside `tsoClient`, it is impossible to strictly distinguish them according to files. The organization logic here is to put as much dispatcher-related code as possible in `tso_dispatcher.go`, and then move some common methods to `tso_client.go`. Perhaps we can further abstract the data structure in the future and put all dispatcher-related logic into a separate structure.